### DISCUSSION TITLE
Aero 7-11 Discussions

### DISCUSSION THREAD

**Discussion by LittleFox2024** [LittleFox2024/Aero7-11#2] [General]
If anyone has any changes they want to see, please let me know, or feel free to make a pull request. I will try to add in as many Windows 7-like features as possible. Thanks!

**Comment by Za9-118** [comment]
The pull requests dont work. I made an Issue that is just like a pull request

**Reply by LittleFox2024** [reply]
OK, I don't know how to use GitHub much so I'm still learning. I'll try to figure that out.

**Comment by Za9-118** [comment]
I'm trying to make a Vista version, i have the shellstyle done, but i cant figure out where the command module background is, and how i can import the control panel sidebar

**Reply by LittleFox2024** [reply]
Are you meaning the msstyle file? It is under CommandModule/MODULEBACKGROUND
![image](https://github.com/user-attachments/assets/bd2c3972-4c92-4b26-8f95-2226b5d30045)

Background I am currently using:
![CommandModule_MODULEBACKGROUND_1183](https://github.com/user-attachments/assets/3d0252c6-1182-476f-baad-13511d785704)

I currently have started but not done much with a Vista style attached here. Feel free to use any of it or edit it directly.
[AeroVista11.zip](https://github.com/user-attachments/files/18129823/AeroVista11.zip)

**Reply by LittleFox2024** [reply]
COMMANDMODULE Properties (They seem to work:)
![image](https://github.com/user-attachments/assets/7b2d37bf-64fc-4443-b73c-d13eab4d2fce)

Also note that if you are using msstyleEditor, when in preview mode the Windows Vista-7 style command bar in File Explorer will always use the classic theme. You must apply the theme through something else (like ThemeTool) in as a full system theme in order to see the effects there.

**Reply by Za9-118** [reply]
I used the one from Aero10, the image is transparent, It broke in testing, but it for some reason somehow fixed itself when i applied it, Anyway, what about the control panel sidebar?

**Reply by Za9-118** [reply]
Also, you have to change the text color values or it looks odd.

**Comment by ghost** [comment]
Hello dear LittleFox, hope its the right lace to address such questions: does your theme support win11 dark mode? its like the only thing keeping me away from installing your theme. since i got my hands on win10 i cant take seriously the light mode. Or maybe your theme is editable with the help of WinPaletter? Thanks in advance.

**Reply by Za9-118** [reply]
Some parts like the window borders will work fine, everything else will look like the Windows 11 dark mode elements. I think that he only edited the light mode parts to look like 7 because Win7 never had a dark theme

**Reply by ghost** [reply]
> Some parts like the window borders will work fine, everything else will look like the Windows 11 dark mode elements. I think that he only edited the light mode parts to look like 7 because Win7 never had a dark theme

You were right sir, im experiencing the only "flaw": if we go into a folder in explorer, in the lowerest section of it it will show folder's icon and how many items are here. So the background of this "section" got some strange brown-ish accent, it happened right after i installed aero 7-11 and checked the dark mode in system settings. Im using default win11 23h2 explorer btw.
![xyixyi](https://github.com/user-attachments/assets/11c813f0-7b66-40ce-9b03-25da2de3eb69)

**Reply by LittleFox2024** [reply]
Sorry about that. That is most likely the issue. I only changed the light mode parts. I do intend to add in dark mode support, but I was planning on getting the light mode finished first. That actually explains quite a bit about another issue a user was having. Thank you for bringing this up.

**Reply by ghost** [reply]
> Sorry about that. That is most likely the issue. I only changed the light mode parts. I do intend to add in dark mode support, but I was planning on getting the light mode finished first. That actually explains quite a bit about another issue a user was having. Thank you for bringing this up.

Here's another screenie for you then:
![xyixyi](https://github.com/user-attachments/assets/5eebf696-9922-4a12-b888-311de267c2f1)
Also i had badly messed up controls from the right corner but ig its because i was using a windhawk script and the dwm blur glass app. And this brown-ish accent again.

**Reply by LittleFox2024** [reply]
I don’t have much experience with WinPalleter. It is based off of the default Windows 11  aero.msstyles, so is should be editable. However, if you would like I can try to add in dark mode support. The main question is do you want the windows 7 style in dark mode, or a dark adaptation of the Windows 7 style? I could of course just put all the elements from the light mode into the dark mode, but that could look weird. Otherwise, it may take a while as I would have to figure out how to adapt the Windows 7 buttons to dark mode.

Although, if there are any specific themes made for Windows 7 with a dark mode that you want me to implement, if you can tell me the theme I can try to do it. If you want to edit it yourself, I use msstylesEditor. However, it can be confusing to work with.

**Reply by Za9-118** [reply]
I think there was a theme in WinClassic that is a dark adaptation of Windows 7, but i forgot the name of it.

**Reply by Za9-118** [reply]
Found it: https://winclassic.net/attachment/download/4977

**Reply by LittleFox2024** [reply]
I think I saw that one. As of now, that is what I think I'll implement.

**Comment by DiscoverTechUniverse** [comment]
The caption buttons are oversized!! How do I get win 7 caption buttons

**Reply by LittleFox2024** [reply]
Use either the small cation button variant (although that has issues) or DWMBlurGlass/OpenGlass to restore the Windows 7 size.

**Reply by DiscoverTechUniverse** [reply]
Well, I tried doing the DWMBlurGlass method, but it doesnt look quite right! In your picture, what did u use to get the Windows 7 caption buttons and which theme did u use?
<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/4f72f9e1-6f4b-4343-bcbc-cbd567b40776" />

**Reply by DiscoverTechUniverse** [reply]
And what version were you using?

**Reply by LittleFox2024** [reply]
If you’re having issues with the button being misaligned you have to edit the config file and add the line `titleBtnOffsetX=7`.  I used the round theme (***not*** the small variant). If theres still issues, can you attach a screenshot with the issue?

**Comment by DiscoverTechUniverse** [comment]
Almost looks 1:1! How do I get accent color title bars, and what DWMBlurGlass settings would you reccomend?
<img width="1051" height="32" alt="image" src="https://github.com/user-attachments/assets/a446085d-f7f3-4aa1-98a4-64d4d2eebdfa" />
@LittleFox2024

**Reply by LittleFox2024** [reply]
Sorry I’ve been a bit slow… I’ll need to check on this tomorrow morning (don’t have my computer with me at the moment). I’ll try to get that first thing.

**Reply by DiscoverTechUniverse** [reply]
Thats alright! I appreciate all your help!

**Reply by LittleFox2024** [reply]
For some reason this wouldn't let me upload `.ini` files... anyway, here's my config file. Remove the `.txt` extension so it's a `.ini` file, then you can go to the `config` tab in DWMBlurGlass to import it. It uses whatever your accent color is and has a pretty weak blur (you can see through quite a bit). It also brings back the Windows 7 style buttons and the glow around them.

This is the default that I use... on the `advanced` tab there is an option to use Accent Color to override RGB. That gets the accent color (it's included in the config file I added).

[AeroConfig.ini.txt](https://github.com/user-attachments/files/22139497/AeroConfig.ini.txt)

**Reply by DiscoverTechUniverse** [reply]
THANK YOU SO MUCH! Its perfect!
<img width="524" height="359" alt="image" src="https://github.com/user-attachments/assets/829c98c9-f7d8-4160-868f-fc4205c4a9d9" />

**Comment by DiscoverTechUniverse** [comment]
How do I bypass startallback trial?

**Reply by LittleFox2024** [reply]
Note I don’t encourage piracy if you can pay for it, and if possible I recommend using Explorer7 instead. But if you have to use this:

Manual way:
[https://gist.github.com/Ashur-D/457562f7aab28bc90babb357f992b26f](https://gist.github.com/Ashur-D/457562f7aab28bc90babb357f992b26f)


Easy way:
[https://github.com/Aetherinox/utility-startallback/tree/main/bin/Release/app.publish](https://github.com/Aetherinox/utility-startallback/tree/main/bin/Release/app.publish)

**Reply by DiscoverTechUniverse** [reply]
Explorer7 on Windows 11 breaks the Settings app!!!!!!!!
